adal/src/main/java/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -615,7 +602,19 @@ public void processRedirectUrl(final WebView view, String url) {
                 // access token
                 new TokenTask(mWebRequestHandler, mAuthRequest, mCallingPackage, mCallingUID)
                         .execute(url);
+                return;
```

Comment:
does this make the code cleaner? Adding an early return in the if to void the else in a non-trivial if block, i don't really understand what is the benefit of dong this.